tensorflow/tools/ci_build/linux/rocm/rocm_py37_pip.sh

```diff
@@ -42,8 +42,18 @@ export N_TEST_JOBS=$(expr ${TF_GPU_COUNT} \* ${TF_TESTS_PER_GPU})
 # Get the default test targets for bazel.
 source tensorflow/tools/ci_build/build_scripts/DEFAULT_TEST_TARGETS.sh
 
+#Add filter tags specific to ROCm version
```

Comment:
Can this be moved to a separate script, so that we can avoid duplicating the logic?